Library/Formula/jq.rb

```diff
@@ -22,7 +22,7 @@ class Jq < Formula
   depends_on "bison" => :build # jq depends on bison > 2.5
 
   def install
-    system "autoreconf", "-iv"
+    system "autoreconf", "-iv" if build.devel? || build.head?
```

Comment:
There is no `devel`, so the first part of this is redundant. If it only applies to `head`, can the autotool dependencies be moved into a `head do` block?